docs/api/session.md

```diff
@@ -1424,23 +1424,35 @@ is emitted.
 Returns `string | null` - The absolute file system path where data for this
 session is persisted on disk.  For in memory sessions this returns `null`.
 
-#### `ses.clearData()`
+#### `ses.clearData([options])`
+
+* `options` Object (optional)
+  * `dataTypes` String[] (optional) - The types of data to clear. If undefined, will clear all types of data. See method details for possible values.
+  * `origins` String[] (optional) - Clear data for only these origins. Cannot be used with `excludeOrigins`.
+  * `excludeOrigins` String[] (optional) - Clear data for all origins except these ones. Cannot be used with `origins`.
+  * `avoidClosingConnections` boolean (optional) - Skips deleting cookies that would close current network connections.
+  * `originMatchingMode` String (optional) - The behavior for matching data to origins. Valid values are `"third-parties-included"` and `"origin-in-all-contexts"`.
 
 Returns `Promise<void>` - resolves when all data has been cleared.
 
-This method clears many different types of data, inlcuding:
+Clears various different types of data.
 
-* Cache
-* Cookies
-* Downloads
-* IndexedDB
-* Local Storage
-* Service Workers
-* And more...
+`dataTypes` may include the following case-sensitive string values:
+
+* **Background Fetch**: `backgroundFetch`
+* **Cache**: `cache`
+* **Cookies**: `cookies`
+* **Downloads**: `downloads`
+* **File Systems**: `fileSystems`
+* **IndexedDB**: `indexedDB`
+* **Local Storage**: `localStorage`
+* **Service Workers**: `serviceWorkers`
+* **WebSQL**: `webSQL`
```

Comment:
For better types, we could put this list of possible values nested in each property like https://www.electronjs.org/docs/latest/api/app#appgetpathname does.

In practice, this might be ugly though since we'd have to duplicate this list 3 times for proper typing unless there's a cute quirky trick in our API JSON generation that can bypass that.